test/dotnet-remove-proj.Tests/GivenDotnetRemoveProj.cs

```diff
@@ -0,0 +1,280 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using FluentAssertions;
+using Microsoft.DotNet.Cli.Sln.Internal;
+using Microsoft.DotNet.Tools.Test.Utilities;
+using System;
+using System.IO;
+using Xunit;
+
+namespace Microsoft.DotNet.Cli.Remove.P2P.Tests
+{
+    public class GivenDotnetRemoveProj : TestBase
```

Comment:
This class and the one above are pretty similar in name in my opinion. I think we should call this one GivenDotnetRemoveProjFromSolution, or better yet, GivenThatWeWantToRemoveProjectsFromASolution.